docker-compose.yml

```diff
@@ -0,0 +1,39 @@
+version: "3.4"
+
+services:
+  localstack-s3:
+    image: localstack/localstack:latest
+    container_name: localstack-s3
+    environment:
+      - SERVICES=s3:5002
```

Comment:
worth to use the [standard `localstack` port for S3](https://github.com/localstack/localstack#overview) if free

Reply:
updated.